Gopkg.lock

```diff
@@ -2,12 +2,12 @@
 
 
 [[projects]]
-  digest = "1:5ad08b0e14866764a6d7475eb11c9cf05cad9a52c442593bdfa544703ff77f61"
+  digest = "1:4d6f036ea3fe636bcb2e89850bcdc62a771354e157cd51b8b22a2de8562bf663"
   name = "cloud.google.com/go"
   packages = ["compute/metadata"]
   pruneopts = "UT"
-  revision = "74b12019e2aa53ec27882158f59192d7cd6d1998"
-  version = "v0.33.1"
+  revision = "c9474f2f8deb81759839474b6bd1726bbfe1c1c4"
+  version = "v0.36.0"
```

Comment:
there are a lot of version changes to other dependencies in the lock file, was that intentional?

Reply:
those are result of `make vendor`

Reply:
That's unexpected to me, I would expect `make vendor` to only update the dependencies that have to do with what changed in the `Gopkg.toml` file, since under the covers `make vendor` simply calls `dep ensure`.  Seeing all these dependencies get updated to newer versions looks more like `make vendor.update` was called instead.

I just tried locally to make the same change as in this PR to the `Gopkg.toml` file:
```
 [[override]]
   name="sigs.k8s.io/controller-tools"
-  source = "https://github.com/ichekrygin/controller-tools.git"
-  revision = "af34042b08d62ea4bf345eabcf2b5a7d9252a4a1"
+  source = "https://github.com/crossplaneio/controller-tools.git"
+  revision = "9acc926ef7ce7372609b34fbafe397158e6442ef"
```

Then I ran `make vendor`.  I only see a single change to the `Gopkg.lock` file for the new version of controller-tools, no other dependencies changed.  Any idea why you saw a lot more changes to the lock file than I did? 🤔

Reply:
nope

Reply:
@jbw976  should be fixed now.